lists/opengl.md

```diff
@@ -30,16 +30,22 @@
 
 - #### [OpenGL 4 Reference](https://www.khronos.org/registry/OpenGL-Refpages/gl4//) [N/A]
 	- Quick reference index for modern OpenGL including version 3.3. Also contains references for GLSL.
+	
+- #### [OpenGL Programming Guide: The Official Guide to Learning OpenGL, Version 4.5 with SPIR-V (9th Edition) ](https://www.amazon.com/OpenGL-Programming-Guide-Official-Learning/dp/0134495497)
+	- Great book for learning OpenGL up to 4.5.
 ### Techniques
 
 
 
 
 
 ### Tutorials
-
-
-
+- #### [OpenGL 3D Game Tutorials](https://www.youtube.com/playlist?list=PLRIWtICgwaX0u7Rf9zkZhLoLuZVfUksDP)
+	- Java OpenGL tutorials, can easily be translated to another programming language.
+- #### [3D Game Development with LWJGL 3](https://lwjglgamedev.gitbooks.io/3d-game-development-with-lwjgl/content/)
+	- LWJGL3 Game Engine tutorial (has source code on GitHub as well).
+- ### [Learn OpenGL](https://learnopengl.com/)
+	- A popular website to get started learning GLFW and OpenGL in C++, however it can easily be translated into other languages.
```

Comment:
I'd rather you drop both Learn "OpenGL" and "3D Game Development with LWJGL 3". I've been informed by other members who contribute to the OpenGL list that they may not be great resources. Drop those and I'll merge the others.